src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,13 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] fleet1 = {new Truck(), new Buldozer(), new Excavator()};
+        for (Machine unit : fleet1) {
```

Comment:
```suggestion
        Machine[] machines = {new Truck(), new Buldozer(), new Excavator()};
        for (Machine machine: machines) {
```
Pay attention when choose names

Reply:
What is wrong with the name? Should we avoid using numbers, or word itself is not understendable?